src/main/resources/api/service_openapi.yaml

```diff
@@ -79,6 +121,34 @@ components:
                 type: array
                 items:
                   type: string
+
+    CreateResourceRequestBody:
+      type: object
+      properties:
+        resourceUid:
+          description: The UID of the resource in CRL format
+          type: object
```

Comment:
this should use our cloud_resource_uid.yaml CloudResourceUid object. Or are you planning on adding that later?

Reply:
Like what I said in previous comments, seems we don't need to depend on CloudResourceUid.yaml. We will parse and convert this in the code?

Reply:
I think we should depend on it. Then our schema says what it should be and in code we get the CloudResourceUid from the request. Maybe we should sync offline.